script.js

```diff
@@ -6,4 +6,244 @@ function showMenu() {
   } else {
     menu.classList.add('display');
   }
-}
\ No newline at end of file
+}
+myObj1 = {
+  id: 0,
+  title: 'tonic',
+  imgSrc: 'elements/snapshoot.png',
+  imgAlt: 'image-project-1',
+  metaDescription: 'A daily selection of privately personalized reads no accounts or sign-ups required.',
+};
+myObj2 = {
+  id: 1,
+  title: 'multi-post stories',
+  imgSrc: 'elements/snapshoot-2.png',
+  imgAlt: 'image-project-2',
+  metaDescription: 'A daily selection of privately personalized reads no accounts or sign-ups required.',
+};
+myObj3 = {
+  id: 2,
+  title: 'tonic',
+  imgSrc: 'elements/snapshoot-3.png',
+  imgAlt: 'image-project-3',
+  metaDescription: 'A daily selection of privately personalized reads no accounts or sign-ups required.',
+};
+myObj4 = {
+  id: 3,
+  title: 'multi-post stories',
+  imgSrc: 'elements/snapshoot-4.png',
+  imgAlt: 'image-project-4',
+  metaDescription: 'A daily selection of privately personalized reads no accounts or sign-ups required.',
+};
+
+const array = [
+  [
+    'tonic',
+    'elements/snapshoot.png',
+    'image-project-1',
+    'A daily selection of privately personalized reads no accounts or sign-ups required.',
+  ],
+  [
+    'multipost-stories',
+    'elements/snapshoot-2.png',
+    'image-project-2',
+    'A daily selection of privately personalized reads no accounts or sign-ups required.',
+  ],
+  [
+    'tonic',
+    'elements/snapshoot-3.png',
+    'image-project-3',
+    'A daily selection of privately personalized reads no accounts or sign-ups required.',
+  ],
+  [
+    'multipost-stories',
+    'elements/snapshoot-4.png',
+    'image-project-4',
+    'A daily selection of privately personalized reads no accounts or sign-ups required.',
+  ],
+];
```

Comment:
- [ ] **OPTIONAL**
Great job managing to code the array of objects, but don't you think it's better when you wrap the objects directly to the array! Please tell me what you think 😄

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
_If you think about it, it'll shorten a lot of lines of code, and less code is better anywhere and anytime. 🎉_